arangod/Replication2/ReplicatedLog/LogFollower.cpp

```diff
@@ -306,6 +354,7 @@ auto replicated_log::LogFollower::appendEntries(AppendEntriesRequest req)
         // It is okay to fire here, because commitToMemoryAndResolve has
```

Comment:
Not since over a year, at least it's now called `checkResultAndCommitIndex`. But however its called, that lambda doesn't even capture the guard anymore since https://github.com/arangodb/arangodb/pull/16035, so the comment is somewhat obsolete.